tensorflow-sys/build.rs

```diff
@@ -47,13 +53,34 @@ fn main() {
         Ok(s) => s == "true",
         Err(_) => false,
     };
+
     if !force_src && env::consts::ARCH == "x86_64" && (env::consts::OS == "linux" || env::consts::OS == "macos") {
         install_prebuilt();
     } else {
         build_from_src();
     }
 }
 
+#[cfg(not(target_env = "msvc"))]
+fn check_windows_lib() -> bool {
+    false
+}
+
+#[cfg(target_env = "msvc")]
+fn check_windows_lib() -> bool {
+    if let Ok(path) = env::var("PATH") {
+        for p in path.split(";") {
+            let path = Path::new(p).join(WINDOWS_LIB);
+            if path.exists() {
+                println!("cargo:rustc-link-lib=dylib={}", LIBRARY);
```

Comment:
As of TensorFlow 1.4, we also need to link in the `tensorflow_framework` library.

Reply:
I have no knowledge on `tensorflow_framework`, but I thought it'd be unnecessary for C API which rust bindings depend on.

> We'd like the C/C++ APIs (libtensorflow.so/libtensorflow_cc.so) to be usable on their own, and those have a linker script so that they don't expose extra symbols (preventing issues like the one you've reported). https://github.com/tensorflow/tensorflow/issues/14267

Does above mean that C API would 'never' use `tensorflow_framework`, or 'optionally'?

If it's optional, I think we should add `framework` feature once framework functionality  implemented on this crate(Is it already done?), and let user choose whether to include `framework` or not. Until that, only with `tensorflow` library seems to be OK.

Also I've noticed pkg-config for `tensorflow` does not include `tensorflow-framework`. Maybe will be included in the future.